src/PDDL-1.0.xml

```diff
@@ -0,0 +1,348 @@
+<spdx name="ODC Public Domain Dedication &amp; License 1.0" identifier="PDDL-1.0" osi-approved="false">
+  <urls>
+    <url>http://opendatacommons.org/licenses/pddl/1.0/</url>
+  </urls>
+  <license>
+    <title>
+      <p>Open Data Commons - Public Domain Dedication &amp; License (PDDL)</p>
+    </title>
+    <optional>
```

Comment:
Agree; I think it should not be optional.

We could reconsider if we empirically find versions of this in the wild that both include and exclude the preamble.